### ISSUE TITLE
家长控制

### ISSUE THREAD

**Issue by yuzhuhoo8** [sirpdboy/openwrt#485]
只对国内的网站有效，国外控制无效

**Comment by sirpdboy** [comment]
是的。目前网址过滤 是这样，因为  科学插件是另外的通道管不了。这不仅是这个插件这样，所有的都这样，包括 应用过滤等 其他 插件也是如此。 目前还没有有效的解决方法 ，管制上机时间，目前家长控制的强力管制 是没问题。如果你有 有好的方法，欢迎提交。
